modules/govuk_jenkins/manifests/job/prune_elasticsearch_snapshots.pp

```diff
@@ -0,0 +1,26 @@
+# == Class: govuk_jenkins::job::prune_elasticsearch_snapshots
+#
+# Delete old Elasticsearch snapshots
+#
+class govuk_jenkins::job::prune_elasticsearch_snapshots (
+    $app_domain = hiera('app_domain'),
+) {
+
+  $check_name = 'prune_elasticsearch_snapshots'
+  $service_description = 'Prune Elasticsearch snapshots'
+  $job_url = "https://deploy.${app_domain}/job/prune_elasticsearch_snapshots"
```

Comment:
This variable isn't used anywhere, it needs to be added to the passive check as:
`action_url => $job_url`